README.md

```diff
@@ -1 +1,72 @@
-# .github
\ No newline at end of file
+
+## Repositories: 
+- Service
+- Purpose
+- Language
+- External Integration
+- Database
+- Infrastructure
+- Main contributor
+
+### 1. ubiquibot/conversation-rewards
+- Service: Conversation Reward Module
+- Purpuse: This is intended to be the proper implementation of comment incentives, based on our learnings from the first go-around.
+- Language: TypeScript
+- External Integration: Github Action, Github API
+- Database: None
+- Main contributor: @pavlovcik
+
+
+### 2. ubiquibot/permit-generation
+- Service: Permit Generation module
+- Purpuse: The purpose of this module is to isolate the bot's payment capabilities. This can be directly invoked from other plugins
+- Language: TypeScript
+- External Integration: ENS, ethers
+- Database: Supabase
+- Main contributor: @Keyrxng, @pavlovcik
+
+### 3. ubiquibot/assistive-pricing
+- Service: Assistive Pricing module
+- Purpose: **Don't know**
```

Comment:
cmd+f in our draft whitepaper for some keywords for a full description https://docs.google.com/document/d/1CtdEVWL_uJ-xvI5qyRIy8pTS7eaqFqSXkB7ldN822dk/edit

Reply:
Assistive pricing is essential for product owners and other managers to instantly set a price for any task. In the current implementation, it is a simple, but fully configurable lookup table between the Priority level and the Time estimate. It also allows for projects to set a “base rate” so that specific projects can have a configurable multiplier to reflect its importance to the organization’s overarching strategic objectives.